### PR TITLE
Feature/bngp 838 av service bus

### PR DESCRIPTION
**Pull request by teddmason**

### CODE REVIEW

File: docker/azure-services/local.settings.json.template [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "AV_API_USER_ID": "",
     "AV_API_BASE_URL": "",
     "AV_API_TOKEN_URL": "",
+    "AV_API_SB_CONNECTION_STRING": "",
     "AzureSignalRConnectionString": "Endpoint=http://azure_services;Port=8888;AccessKey=ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGH;Version=1.0;",
     "WEBSITE_NODE_DEFAULT_VERSION": "~16"
   },
```
Comment (reneli5) [inline docker/azure-services/local.settings.json.template:24]: Did not notice this before, should these details be committed to github

Comment (teddmason) [reply docker/azure-services/local.settings.json.template:24]: Yes AzureSignalRConnectionString is just the dummy/default connection details for the containerised version of signalr we use for local development/testing

**Review by reneli5 — APPROVED** [review]
